Ecosystem Partners/Marketplace Guides/getting-started-with-avni-software-defined-cloud-partner-template.md

```diff
@@ -0,0 +1,153 @@
+{{{
+"title": "Getting Started with Avni Software Defined Cloud - Partner Template",
+"date": "8-13-2015",
+"author": "Bob Stolzberg",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+![logo](http://www.avni.io/media/photologue/photos/cache/Avni-Logo-Tagline-final-20150408_G7m1fR2_730.png)
```

Comment:
I'll get that and fix in the next release, next week.  Right now I wanna get it out.

Thanks,

Bob Stolzberg | Principal Engineer
CenturyLink
Mobile/Text: (314) 397-2610
E-Mail: bob@CTL.io
www.CenturyLinkCloud.com


From: Matthew Osborn <notifications@github.com<mailto:notifications@github.com>>
Reply-To: CenturyLinkCloud/PublicKB <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Fri, 14 Aug 2015 15:10:18 -0700
To: CenturyLinkCloud/PublicKB <PublicKB@noreply.github.com<mailto:PublicKB@noreply.github.com>>
Cc: User <bob.stolzberg@centurylink.com<mailto:bob.stolzberg@centurylink.com>>
Subject: Re: [PublicKB] August Updates (#300)


In Ecosystem Partners/Marketplace Guides/getting-started-with-avni-software-defined-cloud-partner-template.md<https://github.com/CenturyLinkCloud/PublicKB/pull/300#discussion_r37124272>:

> @@ -0,0 +1,153 @@
> +{{{
> +"title": "Getting Started with Avni Software Defined Cloud - Partner Template",
> +"date": "8-13-2015",
> +"author": "Bob Stolzberg",
> +"attachments": [],
> +"contentIsHTML": false
> +}}}
> +
> +![logo](http://www.avni.io/media/photologue/photos/cache/Avni-Logo-Tagline-final-20150408_G7m1fR2_730.png)


I'm skeptical to link off to third party for images... this is called hot linking and is generally frowned upon. Can you down the logo and include it instead?

-
Reply to this email directly or view it on GitHub<https://github.com/CenturyLinkCloud/PublicKB/pull/300/files#r37124272>.

This communication is the property of CenturyLink and may contain confidential or privileged information. Unauthorized use of this communication is strictly prohibited and may be unlawful. If you have received this communication in error, please immediately notify the sender by reply e-mail and destroy all copies of the communication and any attachments.